### ISSUE TITLE
Only loads Level_0

### ISSUE THREAD

**Issue by hteumeuleu** [NicMagnier/PlaydateLDtkImporter#25]
Hi! Thanks for this great library. I've been using it for a new version of [Celeste Classic](https://github.com/hteumeuleu/celeste). It works super well. But when trying to update the framework to the latest version (1.05), it seems like only the first level of my game is loaded correctly. The tilemaps don't get generated for further levels. I'm under the impression that the problem comes form my Ldtk file itself.

Level_0 works fine: 

![Level_0](https://github.com/NicMagnier/PlaydateLDtkImporter/assets/3451753/3f2406d2-6754-48e3-8a39-fccbe92e94e6)

Level_1 (and further levels) don't work: 

![Level_1](https://github.com/NicMagnier/PlaydateLDtkImporter/assets/3451753/84b8f912-ec8b-4142-b572-9ac4789b076d)

Here’s a compiled version (on macOS): 
[Celeste.2.0.0-dev.pdx.zip](https://github.com/NicMagnier/PlaydateLDtkImporter/files/14725875/Celeste.2.0.0-dev.pdx.zip)

(You can use `F` on the keyboard to skip to the next level.)

Here’s the code where I load the map in a `feat/ldtk-update` branch: https://github.com/hteumeuleu/celeste/blob/feat/ldtk-update/Source/Scripts/Objects/Room.lua#L67

Here's the code with an older LDtkImporter version on a `feat/ldtk` branch where every levels work perfectly: https://github.com/hteumeuleu/celeste/blob/feat/ldtk/Source/Scripts/Objects/Room.lua

Thanks for your help!

**Comment by NicMagnier** [comment]
I looked a bit into it and the issue is that the layers and entities are not positioned in the same coordinate space.
From LDtk, the layer position is in world space but the entities in level space.

Unless you have a game that really use world space, I would suggest to position you layers and collisions at 0,0 and not layer.rect.x(and y)

I have no idea what cause the difference between the importer version to be honest. I had a Quick Look and they seem to both calculate the coordinate the same. I suspected a different version of LDtk but also there I didn't find a change that would cause the issue.

For the next version I will add the world_position property to an entity too (since it is now calculated by LDtk)
